doc/tutorials/01_running_blink.md

```diff
@@ -0,0 +1,55 @@
+Running Your First Tock App
+===========================
+
+This guide will help you get the `blink` app running on top of Tock kernel.
+
+
+Setup
+-----
+
+You need to be able to compile and load Tock.
+See the [getting started README](../../README.md) on how to get setup.
+
+You also need hardware that supports Tock.
+
+
+Instructions
+------------
+
+1. **Compile Tock**. In the root of the Tock directory, compile the kernel for
+your hardware platform. You can find a list of boards by running `make list`.
+
+    ```bash
+    cd tock
+    make TOCK_BOARD=imix
+    ```
+
+    This will create binaries of the Tock kernel. Tock is compiled with
+    Cargo, a package manager for Rust applications. The first time Tock is built
+    all of the crates must be compiled. On subsequent builds, crates that haven't
+    changed will not have to be rebuilt and the compilation will be faster.
+
+
+2. **Load the Tock Kernel**. The next step is to program the Tock kernel onto
+your hardware. To do this, run:
+
+    ```bash
+    make TOCK_BOARD=imix program
+    ```
+
+    in the Tock root directory. Now you have the kernel loaded onto the hardware.
+    The kernel configures the hardware and provides drivers for many hardware
+    resources, but does not actually include any application logic. For that, we
+    need to load an application.
```

Comment:
Does it make sense to add a note here saying that if you're an application developer, you'll only need to do this step once?